Designs/CnC.html

```diff
@@ -0,0 +1,90 @@
+
+            On January 21, 2019, French data regulator CNIL imposed a record €50 million fine on Google for breaching the European Union's General Data Protection Regulation. The judgment claimed Google had failed to sufficiently inform users of its methods for collecting data to personalize advertising. Google issued a statement saying it was “deeply committed” to transparency and was “studying the decision” before determining its response.
+
+            On March 20, 2019, the European Commission imposed a €1.49 billion ($1.69 billion) fine on Google for preventing rivals from being able to “compete and innovate fairly” in the online advertising market. European Union competition commissioner Margrethe Vestager said Google had violated EU antitrust rules by “imposing anti-competitive contractual restrictions on third-party websites” that required them to exclude search results from Google's rivals. Kent Walker, Google's senior vice-president of global affairs, said the company had “already made a wide range of changes to our products to address the Commission’s concerns,” and that "we'll be making further updates to give more visibility to rivals in Europe."</p>
+            <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/8/80/Google_bus_protest.jpg/220px-Google_bus_protest.jpg" alt="San Francisco activists protest privately owned shuttle buses that transport workers for tech companies such as Google from their homes in San Francisco and Oakland to corporate campuses in Silicon Valley.">
+        </section>
+    </main>
+    <footer class="bg-light">
+    <div class="row">
+        <div class="col-9" id="references">
+            <h1>References:</h1>
+            <div class="row">
+                <div class="col">
+                    <ol start="269">
+                      <li>"Google censors itself for China". BBC News. January 25, 2006. Archived from the original on November 19, 2018. Retrieved August 4, 2018.</li>
+                      <li>"Google Plans to Launch Censored Search Engine in China, Leaked Documents Reveal · Ryan Gallagher". The Intercept. August 1, 2018. Archived from the original on August 1, 2018. Retrieved August 4, 2018.</li>
+                    </ol>
+                </div>
+                <div class="col">
+                    <ol start="271">
+                      <li>Disis, Jill (September 26, 2018). "Google grilled over 'Project Dragonfly' at Senate hearing on data privacy". CNN. Archived from the original on September 26, 2018.</li>
+                      <li>Gallagher, Ryan (December 17, 2018). "Google's Secret China Project "Effectively Ended" After Internal Confrontation". The Intercept. Archived from the original on March 21, 2019. Retrieved December 17, 2018.</li>
+                    </ol>
+                </div>
+            </div>
+        </div>
+    </div>
+</footer>
```

Comment:
Ive added a footer template into the template.html file
If you can look at that and compare yours i will merge your files

Reply:
All you need to do is add the googleIcon img in and thats it
if you do src="Images/googleIcon.png" it should work since ive got stuff within the master.css file to sort it out
Dont forget an alt attribute